code/modules/admin/verbs/adminhelp.dm

```diff
@@ -188,14 +198,19 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
 
 	statclick = new(null, src)
 	_interactions = list()
+	if(mentor_status)
+		is_mentor_ticket = TRUE
 
 	if(is_bwoink)
 		AddInteraction("<span class='notice'>[key_name_admin(usr)] PM'd [LinkedReplyName()]</span>")
 		message_admins("<span class='notice'>Ticket [TicketHref("#[id]")] created</span>")
+
 	else
 		MessageNoRecipient(msg)
-
-		SSredbot.send_discord_message("admin", "Ticket #[id] created by [usr.ckey] ([usr.real_name]): [name]", "ticket") //send the ticket to discord
+		if(is_mentor_ticket)
+			SSredbot.send_discord_message("admin", "Mentor ticket #[id] created by [usr.ckey] ([usr.real_name]): [name]", "ticket") //send the ticket to discord

```

Comment:
```suggestion
			SSredbot.send_discord_message("mentor", "Mentor ticket #[id] created by [usr.ckey] ([usr.real_name]): [name]", "ticket") //send the ticket to discord
```

New function added into the status cog to check for a `mentor` announcement channel. Mentor tickets will display with a different color embed and will not use `@here` pings if there aren't any admins online.

![image](https://user-images.githubusercontent.com/26130695/60748281-e1255b80-9f51-11e9-88f8-461180f77f33.png)

The mentor channel can be set with `[p]setstatus mentorchannel <channel>`  where [p] is your prefix.

See https://github.com/crossedfall/crossed-cogs/commit/1b3078476a2c4e61cb77f79f062fd22b6afd40b1 for changes.